src/renderer/Core/Settings/Pages/Window/ScrollBehaviour.tsx

```diff
@@ -0,0 +1,37 @@
+import { useSetting } from "@Core/Hooks";
+import { Setting } from "@Core/Settings/Setting";
+import { Dropdown, Option } from "@fluentui/react-components";
+
+export const ScrollBehavior = () => {
+    const scrollBehaviors = {
+        auto: "Auto",
+        smooth: "Smooth",
+        instant: "Instant",
+    };
+
+    const { value: scrollBehavior, updateValue: setScrollBehavior } = useSetting<ScrollBehavior>({
+        key: "window.scrollBehavior",
+        defaultValue: "smooth",
+    });
+
+    return (
+        <Setting
+            label={"Scroll Behaviour"}
```

Comment:
Please use translated labels.

```suggestion
            label={t("scrollBehavior")}
```

Add the translations to `getCoreResources.ts` under `settingsWindow`. Check out `WorkspaceVisibility.tsx` as an example.